packages/wrangler/src/__tests__/metrics.test.ts

```diff
@@ -101,18 +101,18 @@ describe("metrics", () => {
 
 			it("should write a debug log if the request fails", async () => {
 				msw.use(
-					rest.post("*/identify", async (req, res) => {
-						return res.networkError("BAD REQUEST");
+					http.post("*/identify", async () => {
+						return HttpResponse.error();
 					})
 				);
 
 				const dispatcher = await getMetricsDispatcher(MOCK_DISPATCHER_OPTIONS);
 				await dispatcher.identify({ a: 1, b: 2 });
 				await flushPromises();
 				expect(std.debug).toMatchInlineSnapshot(`
-			"Metrics dispatcher: Posting data {\\"type\\":\\"identify\\",\\"name\\":\\"identify\\",\\"properties\\":{\\"a\\":1,\\"b\\":2}}
-			Metrics dispatcher: Failed to send request: request to https://sparrow.cloudflare.com/api/v1/identify failed, reason: BAD REQUEST"
-		`);
+"Metrics dispatcher: Posting data {\\"type\\":\\"identify\\",\\"name\\":\\"identify\\",\\"properties\\":{\\"a\\":1,\\"b\\":2}}
+Metrics dispatcher: Failed to send request: Failed to fetch"
```

Comment:
Hmmm

Reply:
MSW 2 doesn't let you customise the message for network errors (which is where the `reason: BAD REQUEST` came from)